tests/Integration/ScriptsTest.php

```diff
@@ -51,40 +49,23 @@ public function set_up(): void {
 	 * @uses \Parsely\Parsely::api_key_is_set
 	 * @uses \Parsely\Parsely::get_options
 	 * @uses \Parsely\Parsely::update_metadata_endpoint
-	 * @group insert-js
+	 * @group enqueue-js
```

Comment:
Since this script handling is now moved to its own file and own test file, we can probably simplify these `@group` throughout this file down to `scripts`.

Reply:
Agreed.